src/docs/product/accounts/sso/okta-sso/okta-scim.mdx

```diff
@@ -117,11 +127,21 @@ If a user is a member of multiple Okta groups, only the attributes set by the fi
 
 </Alert>
 
+<Alert level="warning">
```

Comment:
i wonder if this should be alert level  "note", as how warning looks in the UI almost says to me that this is a bad thing